.github/workflows/update-helm-charts-index.yml

```
@@ -0,0 +1,56 @@
+name: hashicorp/vault-helm/update-helm-charts-index
+on:
+  push:
+    tags:
+      - 'v[0-9]+\.[0-9]+\.[0-9]+\.*'
```

```suggestion
      - 'v[0-9]+\.[0-9]+\.[0-9]+*'
```
This tag shouldn't have the trailing dot on the last patch version since that makes v#.#.# incompatible